src/screen/DesktopAppDownloadScreen.js

```diff
@@ -0,0 +1,190 @@
+import axios from "axios";
+import styled from "styled-components/native";
+import Ionicons from "@expo/vector-icons/Ionicons";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { Alert } from "react-native";
+import { useState } from "react";
+import { SERVER_PORT } from "@env";
+
+export default function DesktopAppDownloadScreen({ navigation }) {
+  const [text, onChangeText] = useState("");
+  const [isFocus, setIsFocus] = useState(false);
+
+  const logoutAlert = async () => {
+    await AsyncStorage.clear();
+
+    Alert.alert("Logout", "로그아웃이 완료되었습니다.", [
+      {
+        text: "확인",
+        onPress: () => navigation.navigate("Main"),
+      },
+    ]);
+  };
+
+  const validateEmail = async (text) => {
+    const reg = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/;
+
+    try {
+      if (reg.test(text) === false) {
+        Alert.alert("email validate fail", "유효한 이메일이 아닙니다.", [
+          {
+            text: "확인",
+          },
+        ]);
+        onChangeText(text);
```

Comment:
```suggestion
        ]);
        
        onChangeText(text);
```

해당 코드는 2줄 이상의 함수 아래있기 때문에 행간을 띄워주면 한결 보기 편할것같습니다.